### PR TITLE
Added seed data and models for Orders and lineItems

### PR DESCRIPTION
**Pull request by nadiaharrismartinez**

### CODE REVIEW

File: script/seed.js [hunk 1]
```diff
@@ -1,98 +1,264 @@
-'use strict';
+"use strict";
 
+const { TokenExpiredError } = require("jsonwebtoken");
 const {
   db,
   models: { User, Product },
-} = require('../server/db');
+} = require("../server/db");
```
Comment (irais-valenzuela) [inline script/seed.js:7]: Make sure to import your models

File: server/db/models/userProducts.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+
+const userProducts = db.define("userProducts", {
```
Comment (irais-valenzuela) [inline server/db/models/userProducts.js:4]: The variable storing the model needs to be capitalized and singular. Also, the name of model in double quotes needs to be singular. Postgres makes it plural.

File: script/seed.js [hunk 3]
```diff
@@ -1,98 +1,264 @@
...
-        'https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F5e06e150da339f84d1b4b795655e524f485c7679',
+        "https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F5e06e150da339f84d1b4b795655e524f485c7679",
       price: 22.56,
       desc: `Not to throw shade at other succulents, but the string of pearls (Senecio rowleyanus) succulent is a show-stopper. With its overflowing vines and bubbly pearl-like leaves, it’s an eye catcher that looks beautiful trailing across a desk or bookshelf or even hanging in a sunny window, where the vines can trail down for interest. Although these plants look delicate, they’re part of the succulent family—a generally easy-to-care-for group of plants. This one in particular is a sure winner for a special space by your window. Read on for tips for growing and caring for this special plant.`,
     }),
   ]);
 
+  // USER PRODUCTS (PRODUCTS PURCHASED & PRODUCTS IN CART ORGANIZED BY USER)
+  const userProducts = await Promise.all([
+    userProducts.create({
```
Comment (irais-valenzuela) [inline script/seed.js:99]: line 99 and all other creates for userProducts needs to be capitalized. Like this UserProducts (or the name of your model)

File: script/seed.js [hunk 4]
```diff
@@ -1,98 +1,264 @@
...
+      orderId: 6,
+      price: 22.56,
+      qty: 5,
+    }),
+  ]);
+
+  // ALL ORDERS
+  //~~~~> NADIA: Insert user email/shipping addresses when they have been created
+  const order = await Promise.all([
+    order.create({
```
Comment (irais-valenzuela) [inline script/seed.js:140]: line 140 and all other order.creates need to be capitalized Order so you are referring to the model.

File: server/db/models/userProducts.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  productId: {
+    type: Sequelize.INTEGER,
+  },
+  allOrdersId: {
```
Comment (miaolphoebe) [inline server/db/models/userProducts.js:8]: This one should `orderId`, not `allOrdersId`

**Comment by MichelleMullane** [comment]
Not including in main

**Pull request closed without merging**